EDU-2579-EN-Add-new-criteria-behavior-to-Edge-Firewall-Rules-Engine2 (#278)
aziontech/docs: src/content/docs/en/pages/products/edge-firewall/rules-engine.mdx

An Edge Firewall Rule Set can have as many rules as needed. These rules are also interchangeable, as you can share the same rule for all other Edge Firewall configurations.

> > Learn more about [Edge Firewall](/en/documentation/products/edge-firewall/) and its modules.

Reviewer: You can use a button here: Go to Edge Firewall reference

| Request Method                | The request's HTTP method. For example: `GET`, `POST`, `PUT`, etc. | -                        |
| Request URI                   | This relates to the `${uri}` variable from Rules Engine for Edge Applications. The normalized (urldecoded) URI of the request. The value of `${uri}` can change during the processing of a request, for example, when an internal redirect occurs or when index files are used. It doesn't carry the Query String parameters as `${request_uri}` do.| -                        |
| Scheme                        | The scheme of the request: HHTP or HTTPS.                    | -                        |
| Client Certificate Validation | Server process to authenticate client digital certificate. | - | 

> | SSL Verification Status       | Server result to client certificate validation. It can be `SUCCESS`, when the client certificate validation pass; `CERTIFICATE_VERIFICATION_ERROR`, when the client certificate was not valid; and `MISSING_CLIENT_CERTIFICATE`, when the client certificate was not sent in the request. | - |  

Reviewer: ```suggestion
| SSL Verification Status       | Server result to client certificate validation. It can be `SUCCESS`, when the client certificate validation passed; `CERTIFICATE_VERIFICATION_ERROR`, when the client certificate wasn't valid; and `MISSING_CLIENT_CERTIFICATE`, when the client certificate wasn't sent in the request. | - |  
```

Reviewer 2: @peeetry I think is better to use the correct values names. The correct values are `Success`, `Certificate Verification Error` and `Missing Client Certificate`

Author: Great!

| Set WAF Rule Set              | It associates the WAF Rule Set to be used in the request. WAF policies must be previously configured in the *Edge Libraries > WAF Rules*. | Web Application Firewall |
| Run a Function                | It runs a function specified as a parameter. The function must have been previously instantiated and parameterized in the Functions tab in order to be used. | Edge Functions           |

> | Set Custom Response           | Allow a customized response when the request matches the criteria. You can customize the `status_code`, by changing it from 200 to 499; `Content-Type` header; and `body` of your request, with a maximum of 500 characters. | - | 

Reviewer: ```suggestion
| Set Custom Response           | It allows a customized response when the request matches the criteria. You can customize the `status_code` by changing it from 200 to 499, `Content-Type` header, and `body` of your request with a maximum of 500 characters. | - | 
```

Reviewer 2: I suggest to change from `status_code` to `Status Code` and `body` to `Content Body`, like the field names from Edge Firewall Rules Engine
![2023-09-28_15-10](https://github.com/aziontech/docs/assets/54641406/c5db68e7-8384-496d-8484-6d0ebce85394)

Author: Great suggestion, Furno. Thanks!

> The `Maximum Burst size` is only available for the `Req/s` (request per second).

> > Azion platform maintains only one `Set WAF Rule Set` behavior configuration for each criteria logic. If you have two different Edge Firewall Rule Sets configured with the same criteria logic but different `Set WAF Rule Set` behaviors, only the most recent one will be processed. This can be useful in case your application needs constant switching between two or more behaviors for the same criteria.

Reviewer: component: note
